spec/integration/startup-spec.coffee

```diff
@@ -268,6 +268,36 @@ describe "Starting Atom", ->
                 [otherTempDirPath]
               ].sort()
 
+    it "doesn't reopen any previously opened windows if restorePreviousWindowsOnStart is disabled", ->
+      configPath = path.join(atomHome, 'config.cson')
+      config = CSON.readFileSync(configPath)
+      config['*'].core = {restorePreviousWindowsOnStart: false}
+      CSON.writeFileSync(configPath, config)
```

Comment:
What do you think about moving this section before [this LOC](https://github.com/atom/atom/pull/9042/files#diff-a5dc259060c4c3c53c128b8bd0f02d23R285)? This way we make sure that the setting works independently of when we change the configuration.